postman/schemas/index.yaml

```diff
@@ -1,8 +1,13 @@
 openapi: 3.0.0
 info:
   description:
-    An API for simulation and analysis of train traffic on the LEAG rail
-    network
+    A REST API for simulations and analysis of train traffic on the LEAG rail
+    network. Create one of many component configurations, 
+    for example, for defining the interlocking, train schedules, and faults.
+    You can't edit an already specified component configuration but can delete it, including its connections to simulation configurations.
```

Comment:
What happens when you delete it? 
Does the simulation configuration then have a NULL reference or is the deletion cascaded?
Both doesn't sound very desirable 🤔

Reply:
Good question! 
Deleting a component configuration while a connection to a simulation configuration still exists shouldn't be possible, IMO. You have to delete the connections first ;)

Reply:
Or we cascade the deletion. That means to also delete all simulation and run using the deleted item.

Reply:
> Then we delete a connection to the spawner configuration.

That's not allowed 😇 It'll fail.

Reply:
Ok, and what deletion is allowed?
I believe I didn't get it yet 😄

Reply:
I propose: 
- You can delete a component configuration if no connection to a simulation configuration exists.
- You can't delete a connection to a component configuration if at least one run exists for the simulation configuration.
- You can't delete a simulation configuration if a connected run exists. You have to delete the run, including the logs, first. After that, you can delete a simulation configuration, including the connections to the component configuration but not the configuration itself.

Reply:
That sounds good. My inner database-extremist is satisfied :D